worker/src/request.rs

```diff
@@ -204,8 +204,8 @@ impl Request {
     }
 
     /// Access this request's Cloudflare-specific properties.
-    pub fn cf(&self) -> &Cf {
-        self.cf.as_ref().unwrap()
+    pub fn cf(&self) -> Option<&Cf> {
```

Comment:
If we're going to make a breaking change can you update the documentation on why it now returns an `Option`, I think this is a necessary change but I don't want people to be surprised.

Reply:
Sure thing, I updated the docs as well as the examples in the README.

Reply:
@zebp are the documentation updates I did sufficient?